lib/Dialect/TTNN/Pipelines/TTNNPipelines.cpp

```diff
@@ -23,7 +23,7 @@ void createTTNNPipelineTTIRPasses(
 
   // Inlines all private functions. I.e flattens the program into the main
   // function. Removes all private functions.
-  pm.addPass(mlir::createInlinerPass());
+  // pm.addPass(mlir::createInlinerPass());
```

Comment:
@LPanosTT I was having trouble with this pass. Got the following error
```
LLVM ERROR: checking for an interface (mlir::DialectInlinerInterface) that was promised by dialect 'func' but never implemented
```

Is this pass supposed to get registered somewhere? I don't see it in the tablegen files. Only reference I see to it is TTIRInlinerInterface.

Reply:
You’ll need to call mlir::tt::registerAllExtensions() and mlir::func::registerAllExtensions() before you run your passes

Reply:
fantastic, mlir::tt::registerAllExtensions() fixed it. Thank you